apps/frontend/src/libs/components/proggress-bar/progress-bar.tsx

```diff
@@ -0,0 +1,52 @@
+import { Icon } from "~/libs/components/components.js";
+import { PREVIOUS_INDEX_OFFSET } from "~/libs/constants/constants.js";
+import { getValidClassNames } from "~/libs/helpers/helpers.js";
+
+import styles from "./styles.module.css";
+
+type Properties = {
+	currentQuestionIndex: number;
+	totalQuestionsAmount: number;
```

Comment:
We can use this component not only for questions
```suggestion
	currentItemIndex: number;
	totalItemsAmount: number;
```